src/main/java/org/example/home_work5/repository/AccountRepo.java

```diff
@@ -0,0 +1,166 @@
+                AccountDTO accountDTO = createAccountDTO(arrFields);
+
+                result.add(accountDTO);
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        return result;
+    }
+
+    public String creatAccount(AccountDTO account) throws FileNotFoundException {
+
+        // Делаем проверку на уникальность id
+        long accountID = account.getId();
+        boolean idExists = getAccounts().stream().anyMatch(a -> a.getId() == accountID);
+
+        if (idExists){
+            return "Account with id " + accountID + " already exists. Cannot insert duplicate id.";
+        }
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(FILE_PATH, true))){
+            // Преобразование объекта AccountDTO в строку и запись в файл
+            String accountLine = createAccountLine(account);
+            bufferedWriter.write(accountLine);
+            bufferedWriter.newLine();
+
+        } catch (IOException e){
+            throw new RuntimeException(e);
+        }
+
+        return "Recording was successful!";
+    }
+
+    public String updateAccount(long idAccount, AccountDTO newData) throws IOException {
+        List<String> updates = new ArrayList<>();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(FILE_PATH))) {
+            // Пропускаем первую строку с именами столбцов
+            bufferedReader.readLine();
+            String line;
+
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] arrFields = line.split(",");
+                long id = Long.parseLong(arrFields[0].trim());
+
+                if (id == idAccount) {
+                    // Найдена запись для обновления
+                    // Добавляем обновленную
+                    updates.add(createAccountLine(newData));
+                } else {
+                    // Оставляем остальные строки без изменений
+                    updates.add(line);
+                }
+            }
+        } catch (FileNotFoundException e) {
+            return "Error updating account: " + e.getMessage();
+        } catch (IOException e) {
+            return "Error updating account: " + e.getMessage();
+        }
+
+        // Записываем изменения
+        String tempFilePath = "src/main/resources/tempData.csv";
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(tempFilePath))){
+            for (String line: updates){
+                bufferedWriter.write(line);
+                // Каждая запись начинается с новой строки
+                bufferedWriter.newLine();
+            }
+        } catch (IOException e){
+            throw new RuntimeException(e);
+        }
+        // Заменяем исходный файл временным
+        Path sourcePath = Paths.get(tempFilePath);
+        Path destinationPath = Paths.get(FILE_PATH);
```

Comment:
in each operation you rewrite this file, it might be too large for that, so every your CRUD operation may process too long